scripts/scorecards-helm-chart/templates/evaluator.yml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    app.quarkus.io/quarkus-version: 3.10.0
+    app.quarkus.io/commit-id: a06c5574b6a8386827c4a13fb4568f29180834e2
+    app.quarkus.io/vcs-uri: https://github.com/lucamolteni/akrivis.git
+    app.quarkus.io/build-timestamp: 2025-01-17 - 13:58:21 +0000
+  labels:
+    app.kubernetes.io/name: akrivis-evaluator
+    app.kubernetes.io/version: 1.0-SNAPSHOT
+    app.kubernetes.io/managed-by: quarkus
+  name: akrivis-evaluator
+spec:
+  ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: 8083
+  selector:
+    app.kubernetes.io/name: akrivis-evaluator
+    app.kubernetes.io/version: 1.0-SNAPSHOT
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    app.quarkus.io/quarkus-version: 3.10.0
+    app.quarkus.io/commit-id: a06c5574b6a8386827c4a13fb4568f29180834e2
+    app.quarkus.io/vcs-uri: https://github.com/lucamolteni/akrivis.git
```

Comment:
is this for more than testing purposes? fixed URL with `lucamolteni` string

Reply:
so far all the `lucamolteni` strings are mostly repository that are needed to experiment with the deployment as we don't have access to all the registries. Eventually they'll be changed :)